elements/lisk-transactions/test/12_multisignature_transaction.spec.ts

```diff
@@ -418,7 +408,7 @@ describe('Multisignature transaction class', () => {
 		});
 	});
 
-	describe('#addMultisignature', () => {
+	describe.skip('#addMultisignature', () => {
```

Comment:
what are skipping the test? if its unskipped in some other issue can add a TODO here with the issue.

Reply:
We skip this since multi-signature system is changing and these tests will be removed in next epic. Issues should be created for that epic.